docs/owo/recipe-remainders.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Recipe Remainders
+project: owo
+---
+
+This feature allows for recipe specific Remainders within a specific Recipe. Minecraft has an internal Remainder system, but the downside to such is it is global for all recipes 
```

Comment:
> ... recipe specific Remainders within a specific Recipe

This sentence should be made more direct. Just write that the feature allows for Recipe specific Recipe Remainders.

> Minecraft has an internal Remainder system, but the downside to such is it is global for all recipes

This could be a bit cleaner. Something along the lines of `Minecraft has a remainder system for items, although it is restricted to a single remainder per item for all recipes. This system is much more flexible.`

Reply:
> This sentence should be made more direct. Just write that the feature allows for Recipe specific Recipe Remainders.

Agree

> This could be a bit cleaner. Something along the lines of "Minecraft has a remainder system for items, although it is restricted to a single remainder per item for all recipes. This system is much more flexible."

Maybe wording the system as "...designed to be more flexible.", would be better in my eyes.

Reply:
Sounds good to me